core/src/scss/components/button/_button.scss

```diff
@@ -12,12 +12,24 @@
 //
 // Style guide: Simple.Button
 //
+button {
+  &,
+  &.su-button {
+    &,
+    &.su-link {
+      @include button-primary;
```

Comment:
Here as well :) 

@cjwest Do please double check, but I believe we have `<a>` with the `.su-button` and `.su-link` classes, but I don't think we have `<button>` with `.su-link` class.